spec/models/card_test_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+require "helpers"
+
+RSpec.configure do |c|
+  c.include Helpers
+end
+
+RSpec.describe Card,:type => :model do
+  context "card methods tests" do
+    it "Card object with the same original and translated text creation" do
+      c=card_new("mom", "Mom")
+      expect(c.save).to be false
+    end
+
+    it "Actualising review date without DB update" do
+    	expect(card_new("mom", "мама").review_actualize.to_i).to eq DateTime.now.days_since(3).to_i
```

Comment:
Line is too long. [96/80]<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.